release.html

```diff
@@ -0,0 +1,23 @@
+---
+layout: release
+title: Releases
+miniBanner: true
+---
+<h1> OSMT Releases</h1>
+<p class="osmtContent"><em>OSMT Release provided new things added to the OSMT open-source community.</em></p>
+<h2> OSMT 1.0.5 has been released</h2>
+<p><em>RELEASES | John Kallies | OCTOBER 25, 2021</em> </p>
+<p>On behalf of the community, I am pleased to announce that the 1.0.5 release of <a href="https://github.com/wgu-opensource/osmt/releases" target="_blank"> OSMT </a> is available today. The release can be found in <a href="https://github.com/wgu-opensource/osmt/releases" target="_blank">Github</a>. You can check out the 1.0.5 <a href="https://github.com/wgu-opensource/osmt/releases" target="_blank"> release notes </a>for more information.</p>
+<h3> Notable Changes in release 1.0.5:</h3>
+<ul>
```

Comment:
Based on your page layout so far, please make those headers below into h4s, with li's for each bullet

Features and Enhancements: (h4)
- Added capability for multiple alignments
- Allow anonymous API access to searching
- Include BLS job codes when skills are imported

Bug Fixes: (h4)
- Removed ambiguous functionality around author editing 

Development Tooling: (h4)
- Fleshed out repo (license, READMEs, CONTRIBUTING)
- Added unit test coverage
- Aligned docker-compose stacks under a "Quickstart" demo configuration and a "Development" configuration
- Added 'noauth' Spring profile for local development